traffic_ops/testing/api/v14/config/cache_config/cache_config_test.go

```diff
@@ -0,0 +1,221 @@
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cache_config
```

Comment:
Go package should be single word names. Consider `cachecfg` or `cacheconfig`.

See https://golang.org/doc/effective_go.html#package-names